src/data/int/cast.lean

```diff
@@ -119,6 +119,11 @@ begin
   rw [int.mul_div_cancel_left _ this, int.cast_mul, mul_div_cancel_left _ n_nonzero],
 end
 
+@[simp, norm_cast] theorem cast_ite [has_zero α] [has_one α] [has_add α] [has_neg α]
+  (P : Prop) [decidable P] :
+  ∀ m n, ((ite P m n : ℤ) : α) = ite P m n :=
```

Comment:
```suggestion
@[simp, norm_cast] theorem cast_ite [has_zero α] [has_one α] [has_add α] [has_neg α]
  (P : Prop) [decidable P] (m n : ℕ) :
  ((ite P m n : ℤ) : α) = ite P m n :=
```
to match [nat.cast_ite](https://leanprover-community.github.io/mathlib_docs/data/nat/cast.html#nat.cast_ite)